ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/manage/[ingestKeys]/[keyID]/TransformEvent.tsx

```diff
@@ -170,48 +172,60 @@ export default function TransformEvents({ keyID, metadata, keyName }: FilterEven
         />
       </div>
       <div className="mb-6">
-        <CodeBlock
-          tabs={[
-            {
-              label: 'Payload',
-              content: rawTransform ?? defaultTransform,
-              readOnly: false,
-              language: 'javascript',
-              handleChange: handleTransformCodeChange,
-            },
-          ]}
-        />
-      </div>
-      <div className="mb-5 flex gap-5">
-        <div className="w-6/12">
-          <h2 className="pb-1 text-lg font-semibold">Incoming Event JSON</h2>
-          <p className="mb-6 text-sm text-slate-700">
-            Paste the incoming JSON payload here to test your transform.
-          </p>
+        {/* This is needed because Monaco editor collides with Clerk which causes bugs.
+          See https://github.com/clerk/javascript/issues/1643 */}
+        {clerk.loaded && (
           <CodeBlock
             tabs={[
               {
                 label: 'Payload',
-                content: incoming,
+                content: rawTransform ?? defaultTransform,
                 readOnly: false,
-                language: 'json',
-                handleChange: handleIncomingCodeChange,
+                language: 'javascript',
+                handleChange: handleTransformCodeChange,
               },
             ]}
           />
+        )}
+      </div>
+      <div className="mb-5 flex gap-5">
+        <div className="w-6/12">
+          <h2 className="pb-1 text-lg font-semibold">Incoming Event JSON</h2>
+          <p className="mb-6 text-sm text-slate-700">
+            Paste the incoming JSON payload here to test your transform.
+          </p>
+          {/* This is needed because Monaco editor collides with Clerk which causes bugs.
+          See https://github.com/clerk/javascript/issues/1643 */}
+          {clerk.loaded && (
+            <CodeBlock
+              tabs={[
+                {
+                  label: 'Payload',
+                  content: incoming,
+                  readOnly: false,
+                  language: 'json',
+                  handleChange: handleIncomingCodeChange,
+                },
+              ]}
+            />
+          )}
```

Comment:
Git diff is hallucinating here. I only added:

```ts
{/* This is needed because Monaco editor collides with Clerk which causes bugs.
See https://github.com/clerk/javascript/issues/1643 */}
{clerk.loaded && (
// ...
```